_component_design/status-labels.md

```diff
@@ -0,0 +1,150 @@
+{% figure [caption:"Status label within the CTUI project"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/notifications/status-labels/status-labels-hero.png){:width="1440px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+
+## Specifications
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default layout
+
+Status labels are wide tags that include short, brief, and helpful text communication regarding the progress of our users business.
+
+Status labels differentiate themselves from buttons and links by utilizing 100% rounded border radius, in addition to, neutral or alert color strategy which differs from our default buttons and links.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A default status label specification"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/notifications/status-labels/status-labels-default.png){:width="499px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Emphasis
+
+The status label is used to communicate the status of an object or process. You can emphasize or de-emphasize that status data within a UI by modifying the stroke and fill or color property on a status label.
+
+#### Stroke vs fill
+
+- Applying stroke only to a status label de-emphasizes the content while using fill emphasizes the content. 
+- Use all-stroke only status labels when the information contained within is not the most important information on the screen.
+- Conversely, use an all-filled status label variation when the information is of high importance on the screen.
+- If you are using an all-stroke system, you may use fill for a status if you want to draw attention to that status in particular.
+
+#### Color
+
+- Using subdued gray de-emphasizes a status while using emphasis gray or emphasis purple emphasizes a status. 
+- Only use emphasis gray to highlight important statuses since it draws a lot of attention to itself.
+- Only use emphasis purple for a status that is critical for a user to notice since it draws the most attention to itself.
+- Do not use any other color for your status labels unless your are communicating system or device health, in which case you should use a health status label.
+- Take note that on dark backgrounds, lighter colors  (e.g. subdued) will draw more attention than darker colors.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Emphasize and de-emphasize statuses within your system using stroke vs fill and color"] [class:"image bg-light border"] %}
```

Comment:
s/`vs`/`versus`

Also, add a period at the end of a caption as it's a sentence.

Reply:
updated.